staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -418,6 +418,11 @@ const (
 	DeletePropagationForeground DeletionPropagation = "Foreground"
 )
 
+const (
+	// DryRunTrue means that all dry run stages will be processed.
```

Comment:
Which means? Everything but the last etcd storage call? Missing a definition what dry-run stages are.

Reply:
To be honest, we don't know yet what the stages are, and don't really have a proposal for what they might be. I agree that this should be rephrased.

Reply:
What about just writing: DryRunTrue means to execute the complete REST handler with all stages, but don't store the result to etcd.

Reply:
Sounds good to me!